File: model/playlist_tracks.go
package model

import (
	"github.com/jinzhu/gorm"
)

type PlaylistTracks struct {
	gorm.Model
	PlaylistID int64 `gorm: "column: playlist_id; type: bigint; primary_key; auto_increment: false; unique; not null"`
	TrackID    int64 `gorm: "column: track_id; type: bigint; primary_key; auto_increment: false; unique; not null"`
}

File: admin/backend/src/db/model/playlist-tracks.ts
import { Model, snakeCaseMappers } from 'objection';

export class PlaylistTracks extends Model {
    playlistId: string;
    trackId: string;
    
    static get tableName() {
        return 'playlist_tracks';
    }

    static get idColumn() {
        return ['playlist_id', 'track_id'];
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default PlaylistTracks;